pages/k8s/metallb.md

```diff
@@ -42,13 +42,45 @@ explanation of the required configuration</a> from the
 
 </p></div>
 
-Currently, the best way to install MetalLB on Charmed Kubernetes is with a helm chart:
+The best way to install MetalLB in layer 2 mode on Charmed Kubernetes is with 
+the MetalLB Operator. The MetalLB Operator is a charm bundle that allows the 
+deployment of both the controller and speaker components.
+
+To deploy the operator, you will first need a Kubernetes model in Juju.
+Add your Kubernetes as a cloud to your Juju controller:
+
+```
+juju add-k8s k8s-cloud --controller $(juju switch | cut -d: -f1)
+```
+
+And create a new Kubernetes model:
+
+```
+juju add-model metallb-system k8s-cloud
+```
+
+Then deploy the MetalLB operator:
+
+```bash
+juju deploy cs:~charmed-kubernetes/metallb-operator
+```
+
+The IP range allocated to MetalLB can be controlled via the configuration of the metallb-controller:
 
 ```bash
-helm install --name metallb stable/metallb
+juju config metallb-controller iprange=<IP_RANGE>
```

Comment:
can we give an actual example - useful to clarify the delimiter

Reply:
Sure, let me edit that !